src/main/app/case/case-api.ts

```diff
@@ -38,9 +37,6 @@ export class CaseApi {
   }
 
   public async getExistingUserCase(serviceType: string): Promise<CaseWithId | false> {
-    if (config.get('services.case.checkDivCases') && (await this.hasInProgressDivorceCase())) {
-      throw new InProgressDivorceCase('User has in progress divorce case');
```

Comment:
I guess we don't need the InProgressDivorceCase exception class anymore, since we won't throw the exception